x-pack/plugins/cases/public/containers/constants.ts

```diff
@@ -15,6 +15,7 @@ export const casesQueriesKeys = {
   users: ['users'] as const,
   connectors: ['connectors'] as const,
   alerts: ['alerts'] as const,
+  actions: ['user-actions'] as const,
```

Comment:
`actions` is a bit confusing. Maybe we can rename it to `userActions` and rename `userActions` on L29 to `caseUserActions`. Wdyt?

Reply:
yeah, that's better naming.